quickstarts/microsoft.machinelearningservices/machine-learning-job-create-sweep-job/modules/codeversion.bicep

```diff
@@ -0,0 +1,80 @@
+@description('Specifies the name of the Azure Machine Learning workspace where sweep job will be deployed')
```

Comment:
I think these warnings from the Pipeline can be fixed:

```
D:\a\1\s\quickstarts\microsoft.machinelearningservices\machine-learning-job-create-sweep-job\modules\codeversion.bicep(44,9) : Warning use-stable-resource-identifiers: Resource identifiers should be reproducible outside of their initial deployment context. Resource deploymentScript's 'name' identifier is potentially nondeterministic due to its use of the 'utcNow' function (deploymentScript.name -> utcValue (default value) -> utcNow()). [https://aka.ms/bicep/linter/use-stable-resource-identifiers]
D:\a\1\s\quickstarts\microsoft.machinelearningservices\machine-learning-job-create-sweep-job\modules\codeversion.bicep(72,9) : Warning use-stable-resource-identifiers: Resource identifiers should be reproducible outside of their initial deployment context. Resource codeVersionResource's 'name' identifier is potentially nondeterministic due to its use of the 'utcNow' function (codeVersionResource.name -> codeId (default value) -> utcValue (default value) -> utcNow()). [https://aka.ms/bicep/linter/use-stable-resource-identifiers]
D:\a\1\s\quickstarts\microsoft.machinelearningservices\machine-learning-job-create-sweep-job\modules\codeversion.bicep(74,50) : Warning no-hardcoded-env-urls: Environment URLs should not be hardcoded. Use the environment() function to ensure compatibility across clouds. Found this disallowed host: "core.windows.net" [https://aka.ms/bicep/linter/no-hardcoded-env-urls]
```

What do you think?

Reply:
Done.